Source/WebInspectorUI/UserInterface/Views/AuditTestCaseContentView.css

```diff
@@ -36,14 +36,14 @@
     top: 0;
     z-index: var(--z-index-header);
     background-color: var(--audit-test-header-background-color);
-    -webkit-backdrop-filter: blur(20px);
+    -webkit-backdrop-filter: blur(1.25em);
 }
 
 .content-view-container > .content-view.audit-test-case.manager-editing > header h1 > img {
     width: 0.75em;
-    min-width: 12px;
+    min-width: 0.75em;
```

Comment:
`rem` may be better than `em` on most of these, since it's not susceptible to the cascade. In this case, the img in this heading will be a multiple of the base size, times the heading text size, times the image size. The heading size may also be compounded by ancestor `em` sizes making it unpredictable if certain unrelated style changes snuck in. I think changing these to `rem` across the board avoids that unpredictability.